Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -464,6 +464,36 @@ public extension UIView {
 		}
 	}
 
+    /// SwifterSwift: Anchors all sides to superview's safe area.
+    ///
+    /// 1. On iOS 11 and latter anchors view's sides to superview's safe area
+    /// 2. On iOS 9-11 anchors leading and trailing to superview,
+    /// top and bottom to viewController's bottomLayoutGuide and topLayoutGuide.
+    ///
+    /// - Parameter viewController: view controller, of which bottomLahoutGuide
+    ///  and topLayoutGuide are used on iOS 9-11
+    @available(iOS 9, *) public func fillToSuperviewSafeArea(viewController: UIViewController) {
+        guard let superview = self.superview else {
+            fatalError("\(#function) : the view isn't in view hierarchy")
+        }
+
+        translatesAutoresizingMaskIntoConstraints = false
+
+        let leading, top, trailing, bottom: NSLayoutConstraint
+        if #available(iOS 11, *) {
+            leading = self.leadingAnchor.constraint(equalTo: superview.safeAreaLayoutGuide.leadingAnchor)
+            top = self.topAnchor.constraint(equalTo: superview.safeAreaLayoutGuide.topAnchor)
+            trailing = self.trailingAnchor.constraint(equalTo: superview.safeAreaLayoutGuide.trailingAnchor)
+            bottom = self.bottomAnchor.constraint(equalTo: superview.safeAreaLayoutGuide.bottomAnchor)
+        } else {
+            leading = self.leadingAnchor.constraint(equalTo: superview.leadingAnchor)
```

Comment:
Since we notice the there's already a extension `fillToSuperview()` that handles this case, I believe is a good idea not handle this case here, let this extension only handle the safeArea (since it's named `fillToSuperviewSafeArea`) case and also constraint the whole method to iOS 11, so the user can choose to use this the rely on safeArea or the other that we already have. 
@SD10 What you think?

Reply:
@LucianoPAlmeida That makes sense, I'm not opposed to limiting this to iOS 11